diff --git a/node_modules/pastel/build/read-custom-app.js b/node_modules/pastel/build/read-custom-app.js
index a8bb320..394f44a 100644
--- a/node_modules/pastel/build/read-custom-app.js
+++ b/node_modules/pastel/build/read-custom-app.js
@@ -1,5 +1,6 @@
 import fs from 'node:fs/promises';
 import path from 'node:path';
+import {pathToFileURL} from 'node:url';
 export default async function readCustomApp(directory) {
     const files = await fs.readdir(directory);
     let customApp;
@@ -8,7 +9,7 @@ export default async function readCustomApp(directory) {
             continue;
         }
         const filePath = path.join(directory, file);
-        const m = (await import(filePath));
+        const m = (await import(pathToFileURL(filePath).href));
         customApp = m.default;
         break;
     }
